src/items/items.component.tsx

```diff
@@ -1,81 +1,75 @@
 import React from 'react';
-import { Box, Button } from '@mui/material';
-import AddIcon from '@mui/icons-material/Add';
-import ItemDialog from './itemDialog.component';
+import { Box, Button, Grid } from '@mui/material';
 import { useCatalogueItem } from '../api/catalogueItem';
 import { Link, useLocation } from 'react-router-dom';
 import { useCatalogueCategory } from '../api/catalogueCategory';
 import ArrowBackIcon from '@mui/icons-material/ArrowBack';
 
+import ItemsTable from './itemsTable.component';
+
 export function Items() {
-  const [addItemDialogOpen, setAddItemDialogOpen] =
-    React.useState<boolean>(false);
   const location = useLocation();
   const catalogueItemId = location.pathname.split('/')[3];
```

Comment:
Just thought could we use useParams() from react router instead of this? I believe it will use the names inside the paths themselves e.g.
`/catalogue/item/:id` -> `id` (https://reactrouter.com/en/main/hooks/use-params)